### ISSUE TITLE
Inadequate color contrast

### ISSUE THREAD

**Issue by XhmikosR** [twbs/bootstrap#23319]
We have many issues even with WCAG2AA. I think we should try to fix some if not all of the issues.

Some examples I found in our docs homepage:

`.text-muted`
`.btn-outline-secondary`

/CC @patrickhlauke

**Comment by patrickhlauke** [comment]
i tried to tackle most things (including colors) a while ago, but @mdo passed on most of those fixes ... hence https://getbootstrap.com/docs/4.0/getting-started/accessibility/#color-contrast

but sure, now that many changes have landed, there's still many aspects that need to be looked at, including the markup/behavior of widgets like the carousel, tabbed dynamic navigation, accordions, popovers.

**Comment by mdo** [comment]
Yeah, I had passed on a couple PRs because I was planning on replacing most of the existing colors outright. Didn't make sense to optimize those given that :).

**Comment by patrickhlauke** [comment]
As said, now that it's stabilised, I'll give it another crack :)

Also, for sanity's sake, going to keep this issue focused just on color contrast.

**Comment by XhmikosR** [comment]
Yup I'm mostly concerned about color issues too. Please CC me if you make a
patch.

On Aug 11, 2017 22:40, "Patrick H. Lauke" <notifications@github.com> wrote:

As said, now that it's stabilised, I'll give it another crack :)

Also, for sanity's sake, going to keep this issue focused just on color
contrast.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/twbs/bootstrap/issues/23319#issuecomment-321900962>,
or mute
the thread

**Comment by XhmikosR** [comment]
@patrickhlauke: this is not complete. The `a` color for example isn't adequate either.

**Comment by patrickhlauke** [comment]
treat this https://github.com/twbs/bootstrap/issues/25126 as the successor, and feel free to add further examples where we fail (i made a start, but sure with the various shades and variations we use there will be more).

**Comment by XhmikosR** [comment]
Oops, I missed that. I will have a look, thanks!
